lms/djangoapps/courseware/tabs.py

```diff
@@ -332,6 +332,8 @@ def link_func(course, reverse_func):
 
     @classmethod
     def is_enabled(cls, course, user=None):
+        if not super(DatesTab, cls).is_enabled(course, user=user):
+            return False
         """Returns true if this tab is enabled."""
         return RELATIVE_DATES_FLAG.is_enabled(course.id)
```

Comment:
You put the super call above the docstring. And you can just do `super()` without the arguments. Using the arguments is the old python2-style.